docs/scale-and-perf/SLOs.md

```diff
@@ -0,0 +1,41 @@
+# KubeVirt Scalability and Performance SLIs/SLOs
+
+This document is inspired by the [Kubernetes slos document](https://github.com/kubernetes/community/blob/master/sig-scalability/slos/slos.md) and is meant to serve the same purpose but for KubeVirt's control plane.
+
+## KubeVirt Performance and Scale Testing
+The KubeVirt SIG-scale is looking to use two tests to define scale and perf: Burst and Steady State.
+
+The goal for a **Burst Test** is to evaluate the system's performance under heavy load when the user is most
+interested in performing an operation as quickly as possible, without worrying too much about the response time
+but about the time to perform a batch of operations.  A good example of a Burst workload is a sudden spike in
+demand for compute resources to handle a massive increase of users.
+
+The **Burst Test** measures batch VM/VMI creation latency so here's how KubeVirt will test it.
+The Burst Test will ramp up object count in a datacenter, measure, then ramp down object count in a datacenter.
+The measurement will include the Object Creation rate (objects/second), VMI Phase Transition Rate, and a count for
+Kubernetes REST API calls.
+
+The **Steady State Test** takes a different approach than the burst test in that instead of starting the system
+from an unspecified capacity, the Steady State Test focuses on how well the system maintains its starting capacity.
```

Comment:
The **Steady State Test** takes a different approach to the Burst Test, showing the behavior of the system during normal operations under constant load to reach steady state shortly after the test load starts and ramp down to its previous state as soon as as the load is removed. For example, a system that has multiple users generating requests to create, update and delete VMs, in the steady-state phase the system will have an average number of operations per second, for example 20 create, update and delete/s.